Configuration/Applications/python/cmsDriverOptions.py

```diff
@@ -244,13 +244,19 @@ def OptionsFromItems(items):
             raise Exception("Not a valid profiler type %s. Alternatives are pp, mp, fp=<function>."%(profilerType))
 
         options.prefix = "igprof -t cmsRun -%s" % profilerType
-        
+
     if options.heap_profile:
-        if options.profile and options.prefix:
+        if options.heap_profile and options.prefix:
```

Comment:
Repetition of `heap_profile` seems unnecessary (it is already `True`)
```suggestion
        if options.prefix:
```